lib/actions/actions.js

```diff
@@ -17,9 +18,11 @@ class Actions {
     assert(typeof fullMessageText === 'string');
     const commandMap = {
       add: new AddLinkToProjectAction(),
+      overview: new ProjectOverviewAction(),
     };
     if (fullMessageText === '') {
-      throw new TypeError('No Term Found');
+      const actionName = 'overview';
```

Comment:
🙌 Great! Love the idea of a default when one is not found.

Reply:
Thanks!